torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -2031,6 +2031,18 @@ def _backward_impl(ctx, all_args):
                         ),
                     )
 
+                # not ideal but prevent the user from seeing a nasty traceback - See #138422
+                stack = torch._C._functorch.peek_interpreter_stack()
+                vmap_type = torch._C._functorch.TransformType.Vmap
```

Comment:
Change this to raise an exception if something is in the interpreter stack. No functorch tensor have data pointer.